packages/next/export/worker.ts

```diff
@@ -118,7 +118,7 @@ export default async function exportPage({
   httpAgentOptions,
   serverComponents,
 }: ExportPageInput): Promise<ExportPageResults> {
-  setHttpAgentOptions(httpAgentOptions)
+  setHttpClientAndAgentOptions({ httpAgentOptions })
```

Comment:
Is it expected that `experimental` object isn't passed here?

Reply:
Yes because I don't think `exportPage` is ever passed that property.